src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -74,7 +76,7 @@ struct RadixContainer {
   std::shared_ptr<std::vector<bool>> null_value_bitvector;
 };
 
-inline std::vector<size_t> determine_chunk_offsets(std::shared_ptr<const Table> table) {
+inline std::vector<size_t> determine_chunk_offsets(const std::shared_ptr<const Table>& table) {
```

Comment:
Since you did some subtle changes to the hash join, can you please execute a full TPC-H run to evaluate the results?

If you do not have a suitable machine, please tell me and I run the benchmark on a server.

Reply:
https://github.com/hyrise/hyrise/pull/1482#issuecomment-468948284